java/sage/miniclient/MiniClient.java

```diff
@@ -154,12 +163,34 @@ public void write(int b) {}
 
       public synchronized void println(String s)
       {
-        System.err.println(df() + " " + s);
+        if (myProperties.getProperty(redirectLogProp, "false").equalsIgnoreCase("true")){
```

Comment:
You don't need this code in here, because you never use the redir variable if logging is enabled.

Reply:
corrected